app/app.py

```diff
@@ -52,10 +51,18 @@ def odbc_cursor() -> Any:
     """
     import pyodbc
 
-    connection = pyodbc.connect(os.environ["FEATURE_STORE_CONNECTION_STRING"])
+    connection_string = os.environ["FEATURE_STORE_CONNECTION_STRING"]
+
+    if "authentication" not in connection_string.lower():
```

Comment:
should this also be guarded by "if local"
presumably, none of this is required anywhere other than locally?

Reply:
Don't know, but if we only need it in local dev situations, and only test it/use it in those situation, could it cause some strange issues elsewhere? It's probably fine.